PowerPointLabs/PowerPointLabs/ShortcutsLab/ConvertToPicture.cs

```diff
@@ -14,11 +14,14 @@ namespace PowerPointLabs.ShortcutsLab
     internal static class ConvertToPicture
     {
 #pragma warning disable 0618
+        private static int zOrder;
```

Comment:
Can this static var be removed? Maybe change to ConvertToPictureForShape(PowerPoint.Shape shape, int zOrder=0) and pass it in